packages/components/stories/DateTimePicker.js

```diff
@@ -5,6 +5,62 @@ import { IconsProvider } from '../src/index';
 
 import InputDateTimePicker, { DateTimePicker } from '../src/DateTimePickers';
 
+class TestPickerWrapper extends React.Component {
+	static propTypes = {
+		...InputDateTimePicker.propTypes,
+	};
+
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			inputEvalValue: '',
+			selectedDateTime: this.props.selectedDateTime,
+		};
+		this.onSubmitEval = this.onSubmitEval.bind(this);
+		this.onChangeEvalInput = this.onChangeEvalInput.bind(this);
+	}
+
+	onSubmitEval() {
+		this.setState(prevState => ({
+			// eslint-disable-next-line no-eval
+			selectedDateTime: eval(prevState.inputEvalValue),
+		}));
+	}
+
+	onChangeEvalInput(event) {
+		this.setState({
+			inputEvalValue: event.target.value,
+		});
+	}
+	render() {
+		return (
+			<div>
+				<div>
+					<label htmlFor="TestWrapper_storybook_eval-input">
+						Text to eval for updating 'selectedDateTime' prop
+					</label>
+					<br />
+					<input
+						id="TestWrapper_storybook_eval-input"
+						type="text"
+						onChange={this.onChangeEvalInput}
+						value={this.state.inputEvalValue}
+					/>
+					<button onClick={this.onSubmitEval}>Update</button>
+				</div>
+				<br />
+				<InputDateTimePicker
+					{...this.props}
+					selectedDateTime={this.state.selectedDateTime}
+					onChange={action('onChange (event, error message, dateTime)')}
+					onBlur={action('onBlur (event)')}
```

Comment:
`action('onChange')`. The params will appear in storybook logs.
Same for onBlur.
Avoid adding function shape here as it's not maintainable.

Reply:
effectively :)

Reply:
✔